src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Quadrangle {
+    public IsoscelesTrapezoid(String color, int width, int height) {
+        super(color, width, height);
+    }
+
+    @Override
+    public double getArea() {
+        return super.getArea();
+    }
+
+    @Override
+    public String draw() {
+        return "Figure: isoscelesTrapezoid" + super.draw();
+    }
```

Comment:
Don't use abstract classes to set behavior for classes. Since `Quadrangle` is likely an abstract class meant to provide a common base for quadrilateral figures, it should not be used to set behavior. Consider whether `Quadrangle` should be an abstract class or an interface, and if it's an abstract class, ensure it only provides common state, not behavior.